patches/third_party-blink-renderer-core-frame-navigator.idl.patch

```diff
@@ -1,8 +1,13 @@
 diff --git a/third_party/blink/renderer/core/frame/navigator.idl b/third_party/blink/renderer/core/frame/navigator.idl
-index f13febb148e985aa5401ca6ae3221fce45a97261..e8dee21713a439aad1d9edb6aa087d35c7a95192 100644
+index f13febb148e985aa5401ca6ae3221fce45a97261..9c4c7c492de48dc237accdc62ff8dd8c29bbfb56 100644
 --- a/third_party/blink/renderer/core/frame/navigator.idl
 +++ b/third_party/blink/renderer/core/frame/navigator.idl
-@@ -36,7 +36,7 @@
+@@ -32,11 +32,12 @@
+     [HighEntropy, MeasureAs=NavigatorProductSub] readonly attribute DOMString productSub;
+     // https://www.w3.org/Bugs/Public/show_bug.cgi?id=27786
+     [HighEntropy, MeasureAs=NavigatorVendor] readonly attribute DOMString vendor;
++    readonly attribute NavigatorBrave brave;
```

Comment:
pretty sure this isn't needed either - see above note about NavigatorGeolocation